### ISSUE TITLE
The plan to support Kubernetes deployment

### ISSUE THREAD

**Issue by deprecated-awx-xin** [ArroyoSystems/arroyo#51]
Is there any release date plan of this feature?

**Comment by mwylde** [comment]
Yes, Kubernetes support (both as a scheduler for running pipelines and as a deployment target for the Arroyo control plane) is next on our roadmap.

**Comment by akennedy4155** [comment]
Can be converted to a discussion @jacksonrnewhouse @mwylde
